packages/terra-list/src/SingleSelectList.jsx

```diff
@@ -129,7 +136,11 @@ class SingleSelectList extends React.Component {
       newProps.onKeyDown = onKeyDown;
     }
 
+    // Uses the props.hasChevron value, unless the consumer specifies the item hasChevron attribute as false.
     newProps.hasChevron = this.props.hasChevrons;
+    if (item.props.hasChevron === false) {
```

Comment:
I'm referring to the scenario where the value for hasChevrons is false, as the default, and the value for hasChevron is true. Since the item check is only for === false, then true would never be registered.